mage/query-modules/python/import-util.md

```diff
@@ -139,6 +139,83 @@ where `path` is the path to a local JSON file that will be created inside the
 
 </Tabs>
 
+### `graphml(path, config)`
+
+#### Input:
+
+* `path: string` ➡ path to the graphML file that is being imported.
+* `config: Map (default={})` ➡ configuration parameters explained below.
+
+#### Parameters:
+
+| Name 	             | Type   | Default	| Description 	                                                |
+|-	                 |-	      |-	      |-	                                                            |
+| readLabels          | Bool 	| False	  | Create node labels based on the value of the property whose key is "labels". 	|
+| defaultRelationshipType  | String	| "RELATED"  	  | The default relationship type to use when none is specified in the import file. 	|
+| storeNodeIds 	         | Bool 	| False   	| Store node's id attribute as a property.	|
+| source           | Map 	| { }	  | A map that has two keys: label and id. Label is mandatory while id's default value is "id". This allows the import of relationships in case the source node is not present in the file. It will search for a source node with a specific label and a property whose key is this map's id value, and whose value is relationship's source node id. For example, with a config map `{source: {id: 'serial_number', label: 'Device'}}` and an edge defined as `<edge id="e0" source="n0" target="n1" label="CONNECT"><data key="label">CONNECT</data></edge>`, if node "n0" doesn't exist, it will search for a source node `(:Device {serial_number: "n0"})`.  |
+| target     | Map 	| { }	  | A map that has two keys: label and id. Label is mandatory while id's default value is "id". This allows the import of relationships in case the target node is not present in the file. It will search for a target node with a specific label and a property whose key is this map's id value, and whose value is relationship's target node id. For example, with a config map `{target: {id: 'serial_number', label: 'Device'}}` and an edge defined as `<edge id="e0" source="n0" target="n1" label="CONNECT"><data key="label">CONNECT</data></edge>`, if node "n1" doesn't exist, it will search for a target node `(:Device {serial_number: "n1"})`.  	|
```

Comment:
```suggestion
| target     | Map 	| { }	  | A map with two keys: `label` and `id`.  The `label` is mandatory while the `id`'s default value is `id`. This allows the import of relationships in case the target node is absent in the file. It will search for a target node with a specific label and a property equal to the map's `id` value. The value of that property should be equal to the relationship's target node ID. For example, with a config map `{target: {id: 'serial_number', label: 'Device'}}` and an edge defined as `<edge id="e0" source="n0" target="n1" label="CONNECT"><data key="label">CONNECT</data></edge>`, if node "n1" doesn't exist, it will search for a target node `(:Device {serial_number: "n1"})`.  	|
```